### ISSUE TITLE
Update <img> example to also use MP4?

### ISSUE THREAD

**Issue by tigt** [sparkbox/bouncy-ball#58]
[Safari recently allowed the use of `.mp4` video in `<img>` and CSS images.](https://daringfireball.net/linked/2017/12/04/safari-video-gifs) With the `<picture>` source selection that bouncy-ball is already using, this could be as simple as:

```html
<picture>
  <source srcset="image.mp4" type="video/mp4"> <!-- ✨ -->
  <source srcset="image.webp" type="image/webp">
  <source srcset="image.apng" type="image/apng">
  <img src="image.gif" alt="Bouncy ball">
</picture>
```

**Comment by tigt** [comment]
While I’m here, I noticed that the APNG version is about 20 kB smaller than the WEBP one. Their `<source>`s should be swapped so that Firefox gets the smaller version, now that Firefox supports both WEBP and APNG.

**Comment by bryanbraun** [comment]
This was fixed in #59. Closing.
